src/repositories/conversation.ts

```diff
@@ -0,0 +1,64 @@
+import { MessageRepository } from "./message";
+import { Conversation, Message } from "../models";
+import { Repository } from "./repository";
+
+import conversations from "../../data/conversations/data.json";
+
+const messageRepository = new MessageRepository();
+
+// deepcode ignore no-any: JSON
```

Comment:
Delete the comment if not used

Reply:
This comment is for delete typescript error, this isn't useless